internal/otelcollector/config/metric/gateway/config.go

```diff
@@ -14,8 +14,9 @@ type Config struct {
 }
 
 type Receivers struct {
-	OTLP                              config.OTLPReceiver                `yaml:"otlp"`
-	SingletonKymaStatsReceiverCreator *SingletonKymaStatsReceiverCreator `yaml:"singleton_receiver_creator/kymastats,omitempty"`
+	OTLP                              config.OTLPReceiver                 `yaml:"otlp"`
+	SingletonKymaStatsReceiverCreator *SingletonKymaStatsReceiverCreator  `yaml:"singleton_receiver_creator/kymastats,omitempty"`
+	SingletonK8sClusterReceiver       *SingletonK8sClusterReceiverCreator `yaml:"singleton_receiver_creator/k8s_cluster,omitempty"`
```

Comment:
Maybe we rename `SingletonK8sClusterReceiver` to `SingletonK8sClusterReceiverCreator` so that it would be consistent naming with the `SingletonKymaStatsReceiverCreator`?